src/corner_kick/src/components/Canvas/shapes/Rect.tsx

```diff
@@ -0,0 +1,37 @@
+/**
+ * This file defines a Rectangle shape for the Canvas
+ */
+
+import * as React from 'react';
+import { IShape } from 'SRC/types';
+
+interface IRectProps {
+    shape: IShape;
+}
+
+/**
+ * Defines a rectangle that can be displayed in the Canvas
+ */
+export const Rect = (props: IRectProps) => {
+    const { data, fill, stroke, stroke_weight } = props.shape;
+
+    // Fetch data — [x, y, width, height]
```

Comment:
I believe the standard (for graphics stuff) is that `x, y` is the "top left" of the rectangle, right? Would still be good to mention that quickly.

Reply:
Sure! I can add something quickly